src/main/scala/org/broadinstitute/dsde/workbench/sam/service/TosService.scala

```diff
@@ -77,10 +77,7 @@ class TosService (val directoryDao: DirectoryDAO, val registrationDao: Registrat
     */
   def getTosStatus(user: WorkbenchSubject, samRequestContext: SamRequestContext): IO[Option[Boolean]] = {
     if (tosConfig.enabled) {
-      getTosGroupName(samRequestContext).flatMap {
-        case Some(groupName) => directoryDao.isGroupMember(groupName, user, samRequestContext).map(Option(_))
-        case None => IO.raiseError(new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, s"Terms of Service group ${getGroupNameString()} not found.")))
-      }
+      directoryDao.isGroupMember(getGroupName(), user, samRequestContext).map(Option(_))
```

Comment:
The changes in this file are to support this change. We don't need to see if the ToS group exists all the time. It is enough to check when Sam starts and whenever someone accepts or reject the terms. This saves an extra db round trip on every api call.